MapboxAndroidDemo/gradlew

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```

Comment:
Remove this gradlew file and add to .gitignore. I assume this got included by accident since @LornTao looks to be using windows?